src/main/java/ch/uzh/ifi/hase/soprafs24/service/GameDeckService.java

```diff
@@ -384,9 +410,60 @@ public void returnCardsToDealerPile(Game game, String cardsToBeReturned) throws
 
         HttpResponse<String> returnCardsToDealerPileResponse = httpClient.send(returnCardsToDealerPileRequest, HttpResponse.BodyHandlers.ofString());
         logger.info(returnCardsToDealerPileResponse.body());
-
     }
 
+    /**
+     * Allows returning of an explosion card to the dealer pile at an arbitrary location
+     * @param game object in which the card is to be returned
+     * @param location at which the returned explosion card is to be placed
+     * 69 or an integer smaller than -1 -> random location
+     * 0 -> top of the pile
+     * -1 or a location that exceeds the number of remaining cards in the dealer pile returns it to the bottom of the pile
+     *  any other integer > 0 and smaller than the number of remaining cards invokes an exact placement at the desired location.
+     * @param cardToBeReturned
+     * @throws IOException
+     * @throws InterruptedException
+     */
+    public void returnExplosionCardToDealerPile(Game game, Integer location, Card cardToBeReturned) throws IOException, InterruptedException {
+
+        // Random placement in the pile
+        if (location == 69 || location < -1) {
```

Comment:
why 69?

Reply:
lack of creativity :) No, I had to choose a value larger than the deck size of 54 - so 69 seemed like an obvious choice.